db/migrate/20170323225139_add_documents.rb

```diff
@@ -7,7 +7,10 @@ def up
       t.timestamps
     end
 
-    add_attachment :documents, :file
+    add_column :documents, :documents_file_name, :string
+    add_column :documents, :documents_content_type, :string
+    add_column :documents, :documents_file_size, :integer
+    add_column :documents, :documents_updated_at, :datetime
```

Comment:
Yeah I was conflicted about how to approach this. I'll revert the commit for now.